packages/react/runtime/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@preact/signals-react-runtime",
+  "description": "Sub package for @preact/signals-react that contains the pure runtime functions",
+  "private": true,
+  "amdName": "reactSignalsRuntime",
+  "main": "dist/runtime.js",
+  "module": "dist/runtime.module.js",
+  "unpkg": "dist/runtime.min.js",
+  "types": "dist/index.d.ts",
+  "source": "src/index.ts",
+  "exports": {
```

Comment:
I'm guessing you don't plan on this being a private, build-time only package forever, given the full slew of package formats?

Reply:
The sub packages of preact (e.g. [preact/hooks](https://github.com/preactjs/preact/blob/master/hooks/package.json)) have all formats also defined so I did the same.

I haven't fully decided on if this'll be its own package but its not out of the question.

Reply:
Ah, fair enough!

For some reason I was thinking that this would be inlined into the React package. My bad!